app/components/pages/SubmissionWizard/steps/Files/index.js

```diff
@@ -113,107 +117,128 @@ export const onFileDrop = (
-                      {removeSupportFiles => (
-                        <SupportingUpload
-                          data-test-id="supportingFilesUpload"
-                          files={submissionFiles.filter(
-                            file => file.type === SUPPORTING_FILE,
+          return (
+            <Mutation mutation={DELETE_MANUSCRIPT_MUTATION}>
+              {deleteFile => (
+                <React.Fragment>
+                  <Box mb={3} width={1}>
+                    <ValidatedField
+                      component={CoverLetterEditor}
+                      id="coverLetter"
+                      name="coverLetter"
+                      onBlur={value => setFieldValue('coverLetter', value)}
+                      onChange={value => setFieldValue('coverLetter', value)}
+                    />
+                  </Box>
+                  <Box mb={3} width={1}>
+                    <ManuscriptUpload
+                      conversion={{
+                        converting: loading || uploadData.uploadProgress < 100,
+                        completed: hasManuscript,
+                        progress: getProgress(uploadLoading, uploadData),
+                        error: uploadError,
+                      }}
+                      data-test-id="upload"
+                      fileName={manuscriptFile.filename}
+                      formError={
+                        touched[filesFieldName] && errors[filesFieldName]
+                      }
+                      onDrop={files =>
+                        onFileDrop(
+                          files,
+                          {
+                            setFieldTouched,
+                            setFieldError,
+                            setFieldValue,
+                          },
+                          deleteFile,
+                          uploadFile,
+                          values.id,
+                        )
+                      }
+                    />
+                  </Box>
+                  <Box width={1}>
+                    <Mutation mutation={UPLOAD_SUPPORTING_MUTATION}>
+                      {uploadSupportFiles => (
+                        <Mutation mutation={DELETE_SUPPORTING_FILES_MUTATION}>
+                          {removeSupportFiles => (
+                            <SupportingUpload
+                              data-test-id="supportingFilesUpload"
+                              files={submissionFiles.filter(
+                                file => file.type === SUPPORTING_FILE,
+                              )}
+                              hasManuscript={hasManuscript}
+                              removeFiles={() =>
+                                removeSupportFiles({
+                                  variables: { id: values.id },
+                                })
+                              }
+                              uploadFile={file => {
+                                this.props.client.readFragment({
```

Comment:
This doesn't look complete yet, lets remove and just include `graphql` query changes